vmdb/app/views/shared/views/_compliance_history.html.haml

```diff
@@ -0,0 +1,18 @@
+#compliance_history_div
+  = render :partial => "layouts/flash_msg"
+  - squash_state = session[:squash_open] && session[:squash_open] == true ? "false" : "true"
+  - squash_title = session[:squash_open] && session[:squash_open] == true ? _("Collapse All") : _("Expand All")
```

Comment:
do we need an explicit match with `true` here? or would `session[:squash_open] ? ....` be ok?